src/mozxchannel/git/process.py

```diff
@@ -304,10 +312,12 @@ def createWorkdir(self, contents):
         self.ensureL10nToml(workdir)
 
     def ensureL10nToml(self, workdir):
-        includes = {
-            '{}/l10n.toml'.format(repo.target_root)
-            for repo in self.graph.repos
-        }
+        includes = set()
+        for repo in self.graph.repos:
+            # TODO Support focus-android and fenix
+            folder = "{}/android-component".format(repo.target_root) if "firefox-android" in repo.target_root else repo.target_root
```

Comment:
Should this be "android-component**s**" with an `s`?
Obligatory hardcode comment.

One thought: we could detect monorepo somehow (single-location hardcode for firefox-android or staging-firefox-android) and then create a config map, e.g.

```
path_rewrite_subdirectory = ""
if monorepo:
    path_rewrite_subdirectory = "/android-components"

folder = f"{repo.target_root}{path_rewrite_subdirectory}"
```

That could allow us to go both ways, programmatically, where the hardcodes only happen for a top-level config dict?

Also, nit: f-strings?

Reply:
> Should this be "android-components" with an s?

Great catch! This bug actually impacted this PR: https://github.com/mozilla-l10n/android-l10n/pull/516/files#diff-ce988e742b10a22bdaf4857b59f80e15b5ebc3641948fb352d2e5386ef4d507bR10. That said, I'm not sure the impact it had in the long rune. 

> One thought: we could detect monorepo somehow (single-location hardcode for firefox-android or staging-firefox-android)

Good idea! I'm not sure if I'd like to go that route. These hacks are meant to be temporary. Making bigger changes in `mozxchannel` seems risky to me. 

> Also, nit: f-strings?

The rest of the code base still uses `.format()`. I stuck to it. It won't be too much of a problem notably because this code will eventually be reverted.